### ISSUE TITLE
unable to download by given commands

### ISSUE THREAD

**Issue by deepjenya** [transcriptionstream/transcriptionstream#15]
ERROR: failed to solve: process "/bin/sh -c pip install -r requirements.txt" did not complete successfully: exit code: 2
Creating Docker volumes...
transcriptionstream
Starting services with docker-compose...
[+] Running 4/4
 ✘ init Error                                                                                                                                                                              4.1s
 ✘ ts_meilisearch_service Error                                                                                                                                                            4.1s
 ✘ ts_transcription_service Error                                                                                                                                                          4.1s
 ✘ ts_gpt_service Error                                                                                                                                                                    4.1s
Error response from daemon: pull access denied for ts-gpu, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
Downloading  transcriptionstream mistral model
curl: (28) Failed to connect to 172.30.1.3 port 11434 after 21039 ms: Couldn't connect to server

How to overcome this?

**Comment by transcriptionstream** [comment]
Can you provide any more detail? What type of system are you using? What script did you run?

**Comment by Joshfindit** [comment]
Looks like this is due to compose getting confused.

Since there is no `build` context in the compose file, if there is not already a local image called `ts-gpu` docker will attempt to find it on dockerhub. `transcriptionstream/ts-gpu` exists, but `ts-gpu` does not.

**Comment by stl314159** [comment]
I think your install.sh errored out but the script goes ahead and tries to `docker compose up` anyway.  The `pull access denied` is a red herring.  You need to scroll back in your logs and find what caused the docker build to error.

**Comment by transcriptionstream** [comment]
Yeah - not a lot of error checking in the install script. A manual build or utilizing the no-build script would be a better choice if you're having issues.
